Gecco convert (#506)
nf-core/funcscan: docs/output.md

[deepBGC](https://github.com/Merck/deepbgc) detects BGCs in bacterial and fungal genomes using deep learning. DeepBGC employs a Bidirectional Long Short-Term Memory Recurrent Neural Network and a word2vec-like vector embedding of Pfam protein domains. Product class and activity of detected BGCs is predicted using a Random Forest classifier.

> #### GECCO & GECCO CONVERT

Reviewer: [![suggestion](https://img.shields.io/badge/suggestion-3B82F6)](https://pullpo.io/cc?l=suggestion) 
```suggestion
#### GECCO
```

They are from the same package so should be fine as just  GECCO

<details markdown="1">
<summary>Output files</summary>

- `gecco/`
  - **GECCO**
  - `*.genes.tsv/`: TSV file containing detected/predicted genes with BGC probability scores
  - `*.features.tsv`: TSV file containing identified domains
  - `*.clusters.tsv`: TSV file containing coordinates of predicted clusters and BGC types
  - `*_cluster_*.gbk`: GenBank file (if clusters were found) containing sequence with annotations; one file per GECCO hit

  - **GECCO CONVERT**
  - `*.gff`: GFF3 converted cluster tables containing the position and metadata for all the predicted clusters
  - `*.region*.gbk`: Converted and aliased GenBank files so that they can be loaded by BiG-SLiCE
  - `*.faa`: Amino-acid FASTA converted GenBank files of all the proteins in a cluster
  - `*.fna`:Nucleotide sequence FASTA converted GenBank files from the cluster

>   **ONLY IF --run_gecco_convert**

Reviewer: [![suggestion](https://img.shields.io/badge/suggestion-3B82F6)](https://pullpo.io/cc?l=suggestion) 
```suggestion
  - **GECCO CONVERT**
  - `*.gff`: GFF3 converted cluster tables containing the position and metadata for all the predicted clusters (only if `--bgc_gecco_runconvert`)
  - `*.region*.gbk`: Converted and aliased GenBank files so that they can be loaded by BiG-SLiCE (only if `--bgc_gecco_runconvert`)
  - `*.faa`: Amino-acid FASTA converted GenBank files of all the proteins in a cluster (only if `--bgc_gecco_runconvert`)
  - `*.fna`:Nucleotide sequence FASTA converted GenBank files from the cluster (only if `--bgc_gecco_runconvert`)
```

For the change of parameter name, see comment below.

  </details>

[GECCO](https://gecco.embl.de) is a fast and scalable method for identifying putative novel Biosynthetic Gene Clusters (BGCs) in genomic and metagenomic data using Conditional Random Fields (CRFs).

> [GECCO CONVERT] (https://gecco.embl.de) is an option in gecco which does file conversion into formats like GFF3, GenBank, or FASTA for further analysis.

Reviewer: ```suggestion
The additional  GFF3, GenBank, or FASTA files from `--bgc_gecco_runconvert`, can be useful for additional further analysis of the BGC hits.
```

<summary>Output files</summary>

- `gecco/`
  - **GECCO**
  - `*.genes.tsv/`: TSV file containing detected/predicted genes with BGC probability scores
  - `*.features.tsv`: TSV file containing identified domains
  - `*.clusters.tsv`: TSV file containing coordinates of predicted clusters and BGC types
  - `*_cluster_*.gbk`: GenBank file (if clusters were found) containing sequence with annotations; one file per GECCO hit

>   - **GECCO CONVERT**

Reviewer: ```suggestion
```

[GECCO](https://gecco.embl.de) is a fast and scalable method for identifying putative novel Biosynthetic Gene Clusters (BGCs) in genomic and metagenomic data using Conditional Random Fields (CRFs).

> #### GECCO CONVERT

Reviewer: This should be the merged with the section as above @SkyLexS as `convert` is just a subcommand of `gecco`, but otherwise the contents is good :)

When merging, you can maybe put at the end of the bullet points you've added something liek '(only if `--run_convert`)' is given to show that it is optional

Author: is something like this okay?<img width="1431" height="396" alt="image" src="https://github.com/user-attachments/assets/c2199b70-2830-4af3-842c-f0562afa8012" />
